src/test/java/hudson/plugins/emailext/EmailExtStepTest.java

```diff
@@ -0,0 +1,28 @@
+package hudson.plugins.emailext;
+
+import org.jenkinsci.plugins.workflow.steps.StepConfigTester;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.recipes.WithPlugin;
+
+/**
+ * Created by acearl on 9/15/2015.
+ */
+public class EmailExtStepTest {

```

Comment:
Good idea to add a test actually running the step once, using `mock_javamail` I suppose.